spec/druuid_spec.rb

```diff
@@ -1,37 +1,42 @@
 require_relative '../druuid'
 
 describe Druuid do
-  describe '.gen' do
+  shared_examples "druuid generator" do |method|
```

Comment:
Whoops, this should be getting called with both `:gen` and `:min_for_time`. Fixed that and had to move the uniqueness test into the `describe '.gen'` context since a call to `Druuid.min_for_time` won't produce a unique value.